rclcpp/cmake/rclcpp_create_node_main.cmake

```diff
@@ -0,0 +1,28 @@
+# Copyright 2015 Open Source Robotics Foundation, Inc.
```

Comment:
This file was just moved (in the first commit) therefore it doesn't need to be updated.